doc/services/portability/posix/option_groups/index.rst

```diff
@@ -85,8 +85,8 @@ This table lists service support status in Zephyr:
     pthread_attr_getstack(),yes
     pthread_attr_setguardsize(),
     pthread_attr_setstack(),yes
-    pthread_getconcurrency(),
```

Comment:
so, IMO you need to document this differently here, maybe as posix implementation continues to grow we need a better way to document specifics, not a table.

Reply:
Correct. I brought the same thing up in my previous PR that addressed POSIX docs. It's out of scope of the current PR though.

Reply:
sure, I said "maybe" indicating this is something to look into in the future. My point here is that this change to the docs is still wrong.